src/operator/tensor/dot-inl.cuh

```diff
@@ -573,12 +581,113 @@ inline void DotCsrDnsDnsImpl(const OpContext& ctx,
                   data_out.dptr<DType>(), data_l.dptr<DType>(), indptr_l.dptr<IType>(),
```

Comment:
Since DotCsrTransDnsDnsWarpKernel, DotCsrTransDnsDnsThreadBlockKernel, and DotCsrTransDnsDnsWarpBlockKernel are nondeterministic and cannot be used, we might as well just remove them.

Reply:
Sure I can do that.

Reply:
Done.